collect_app/src/main/java/org/odk/collect/android/instancemanagement/autosend/InstanceAutoSendFetcher.kt

```diff
@@ -1,33 +1,31 @@
 package org.odk.collect.android.instancemanagement.autosend
 
-import android.app.Application
+import org.odk.collect.forms.Form
 import org.odk.collect.forms.FormsRepository
 import org.odk.collect.forms.instances.Instance
 import org.odk.collect.forms.instances.InstancesRepository
-import org.odk.collect.settings.SettingsProvider
-import org.odk.collect.settings.enums.AutoSend
-import org.odk.collect.settings.enums.StringIdEnumUtils.getAutoSend
 
 object InstanceAutoSendFetcher {
 
     fun getInstancesToAutoSend(
-        application: Application,
         instancesRepository: InstancesRepository,
         formsRepository: FormsRepository,
-        settingsProvider: SettingsProvider
+        formAutoSend: Boolean = false
```

Comment:
Because it gets called in two cases:

1. We are doing a regular auto send triggered by sending a form without auto send enabled at a form level
2. We are doing an auto end triggered by a form with auto send enabled and just want to send the "opt-in" forms

The two different cases have different network constraints. I guess we technically could send the form level auto send `true` forms with the other ones if they're waiting to send, but I found it simpler to separate the logic out entirely.